lib/pkgcloud/openstack/compute/client/servers.js

```diff
@@ -275,18 +275,44 @@ exports.rebootServer = function rebootServer(server, options, callback) {
  *
  * @description Rebuild the provider server
  *
- * @param {String|object}   server      The server or serverId to rebuild
- * @param {String|object}   image       The image or imageId to use in the rebuild
+ * @param {String|object}   server               The server or serverId to rebuild
+ * @param {String|object}   options              The image or imageId to use in the rebuild (for backwards compatibility)
+ * @param {String|object}   options.image        The image or imageId to use in the rebuild
+ * @param {String|object}   options.flavor       The flavor or flavorId to use in the rebuild
+ * @param {String}          options.accessIPv4   The IP version 4 address of the server.
+ * @param {String}          options.accessIPv6   The IP version 6 address of the server.
+ * @param {String}          options.adminPass    The administrator password for the server.
+ * @param {object}          options.metadata     Metadata key/value pairs.
+ * @param {array}           options.personality  Personality files - path and contents
+ * @param {String}          options.personality[n].path      Personality file path
+ * @param {String}          options.personality[n].contents  Personality file contents
+ * @param {String}          options['OS-DCF:diskConfig']    The disk configuration value ("AUTO" | "MANUAL").
  * @param {Function}        callback
  * @returns {*}
  */
-exports.rebuildServer = function rebootServer(server, image, callback) {
-  var imageId = image instanceof base.Image ? image.id : image;
+exports.rebuildServer = function rebootServer(server, options, callback) {
```

Comment:
Ah. I like this. I was thinking of just replacing `server` and `image` with `options`, but this makes more sense. Well done.